test/e2e/installplan_e2e_test.go

```diff
@@ -3122,6 +3217,18 @@ var _ = Describe("Install Plan", func() {
 		_, err = crc.OperatorsV1().OperatorGroups(ns.GetName()).Create(context.TODO(), og, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
+		// Wait for the OperatorGroup to be synced and have a status.ServiceAccountRef
+		// before moving on. Otherwise the catalog operator treats it as an invalid OperatorGroup
+		// and fails the InstallPlan
+		Eventually(func() (bool, error) {
+			outOG, err := crc.OperatorsV1().OperatorGroups(ns.GetName()).Get(context.TODO(), og.Name, metav1.GetOptions{})
+			if err != nil {
+				return false, err
+			}
+			fmt.Fprintf(GinkgoWriter, "[DEBUG] Operator Group Status: %+v\n", outOG.Status)
```

Comment:
Similar comment here, the test context has a Logf interface that does approximately the same thing.

Reply:
Forgot I had two of these blocks. Coming in the next update.